sites/partners/src/components/listings/PaperListingForm/sections/RankingsAndResults.tsx

```diff
@@ -119,6 +119,32 @@ const RankingsAndResults = ({ listing }: RankingsAndResultsProps) => {
         )}
         {reviewOrder === "reviewOrderLottery" && (
           <>
+            {process.env.showLottery && (
+              <Grid.Row columns={2} className={"flex items-center"}>
+                <Grid.Cell>
+                  <p className={`field-label m-4 ml-0`}>{t("listings.lotteryOptInQuestion")}</p>
+                  <FieldGroup
+                    name="lotteryOptInQuestion"
+                    type="radio"
+                    register={register}
+                    fields={[
+                      {
+                        ...yesNoRadioOptions[0],
+                        id: "lotteryOptInYes",
+                        defaultChecked:
+                          !listing || listing.lotteryOptIn === true || !listing.lotteryOptIn,
+                      },
+
+                      {
+                        ...yesNoRadioOptions[1],
+                        id: "lotteryOptInNo",
+                        defaultChecked: listing && listing.lotteryOptIn === false,
```

Comment:
Should be `!listing`

Reply:
If listing is false / null, we want to default to yes